docs/2.9. Usage - Fully Automated Testing of Controllers.md

```diff
@@ -0,0 +1,180 @@
+
+```json
+{
+    "test_type": "radio",
+    "question_id": "test_05_1",
+    "name": "test_05",
+    "description": "Reference Sender is put offline and then back online",
+    "question": "Please refresh your NCuT and select the sender which has been put 'offline'",
+    "answers": [
+        {
+            "answer_id": "answer_2",
+            "label": "s3/wright",
+            "description": "Mock sender 3",
+            "id": "71ce775c-cdc5-479a-bd29-87c081905ecf",
+            "answer_str": "s3/wright (Mock sender 3, 71ce775c-cdc5-479a-bd29-87c081905ecf)"
+        },
+        {
+            "answer_id": "answer_3",
+            "label": "s4/mason",
+            "description": "Mock sender 4",
+            "id": "14db3fcd-3606-4679-9c2a-252b4af1e611",
+            "answer_str": "s4/mason (Mock sender 4, 14db3fcd-3606-4679-9c2a-252b4af1e611)"
+        },
+        {
+            "answer_id": "answer_4",
+            "label": "s5/barrett",
+            "description": "Mock sender 5",
+            "id": "b0a5f961-21de-4ec7-951e-58bbea8934f3",
+            "answer_str": "s5/barrett (Mock sender 5, b0a5f961-21de-4ec7-951e-58bbea8934f3)"
+        }
+    ],
+    "time_sent": 1630580530.0981488,
+    "timeout": 600,
+    "url_for_response": "http://127.0.0.1:5000/testingfacade_response",
+    "answer_response": "",
+    "time_answered": "",
+    "metadata": null
+}
+```
+
+Example action question POST request from Test Suite to '/x-nmos/controller-tests' including metadata:
+
+```json
+{
+    "test_type": "action",
+    "question_id": "test_07",
+    "name": "test_07",
+    "description": "Instruct Receiver to subscribe to a Sender's Flow via IS-05",
+    "question": "All flows that are available in a Sender should be able to be connected to a Receiver. Use the NCuT to perform an 'immediate' activation between sender: s5/barrett (Mock sender 5, 490dc37e-eb1c-45f8-ac1c-364f15335d49) and receiver:r6/gilliam (Mock receiver 6, f3a243b0-3699-409b-ae1d-e260e2a29e2f) Click the 'Next' button once the connection is active.",
+    "answers": [],
+    "time_sent": 1630658951.439078,
+    "timeout": 600,
+    "url_for_response": "http://127.0.0.1:5000/testingfacade_response",
+    "answer_response": "",
+    "time_answered": "",
+    "metadata": {
+        "sender": {
+            "id": "490dc37e-eb1c-45f8-ac1c-364f15335d49",
+            "label": "s5/barrett",
+            "description": "Mock sender 5"
+        },
+        "receiver": {
+            "id": "f3a243b0-3699-409b-ae1d-e260e2a29e2f",
+            "label": "r6/gilliam",
+            "description": "Mock receiver 6"
+        }
+    }
+}
+```
+
+An example class for storing and retrieving this data can be found at 'testingfacade/DataStore.py'
+
+On receiving a POST request from the Testing Tool, your application should use the question id to instruct your chosen automated test framework (eg. Selenium) to carry out the required tasks on your Controller and return appropriate answers.
```

Comment:
Could the word 'return' be made clearer here?

When I first read it, I thought this meant the response to the `POST` request... Reading on, I believe that the `POST` request should be responded to immediately (with a [`202 Accepted`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202) noncommittal response?) and then a `POST` request made from the app back to the testing tool on the `url_for_response`, right?